frontend/src/metabase/browse/databases/BrowseDatabases.tsx

```diff
@@ -34,7 +48,7 @@ export const BrowseDatabases = () => {
     return <LoadingAndErrorWrapper loading />;
   }
 
-  if (!databases?.length) {
+  if (!databases?.length && !isAdmin) {
```

Comment:
This was probably the "biggest" conceptual change in this whole PR.
Even when there are no databases, we now want to display the new "Add database" card in order to prompt admins to add a new database.

Empty state still shows for non-admins, though.